tests/unittests/CoreText/CTFrameTests.mm

```diff
@@ -25,7 +25,7 @@
 static constexpr float c_arbitraryFloat = 314159.2717;
 
 static NSAttributedString* getAttributedString(NSString* str) {
-    UIFontDescriptor* fontDescriptor = [UIFontDescriptor fontDescriptorWithName:@"Times New Roman" size:40];
+    UIFontDescriptor* fontDescriptor = [UIFontDescriptor fontDescriptorWithName:@"Segoe UI" size:40];
```

Comment:
The test values were done using Segoe, so when we could actually use Times it threw off all of the numbers.  tl;dr it's a lazier fix than adjusting all of the values for a different font unrelated to the test

---
In reply to: [79930359](https://github.com/Microsoft/WinObjC/pull/1021#discussion_r79930359) [](ancestors = 79930359)